index.js

```diff
@@ -98,7 +99,13 @@ function isRoot(dir) {
 }
 
 function getSourceDirectories(elmPackagePath) {
-  var elmPackage = JSON.parse(fs.readFileSync(elmPackagePath, 'utf8'));
+  try {
+    var elmPackage = JSON.parse(fs.readFileSync(elmPackagePath, 'utf8'));
```

Comment:
maybe we should also check if `elmPackage` has a `source-directories` field, otherwise it will crash on line 109